src/icons/arrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
```

Comment:
Don't we already have a built-in set of icons to use?

Reply:
we have couple of arrow icons, they look this way
![image](https://user-images.githubusercontent.com/44117648/120198553-d9ef5100-c23f-11eb-89f4-05e7b83e9658.png)
or
![image](https://user-images.githubusercontent.com/44117648/120198634-f4292f00-c23f-11eb-8a3a-eee87c1eb157.png)

If we already have something similar to [this](https://github.com/parse-community/parse-dashboard/pull/1708#issuecomment-850977855), could you please redirect me to that file.

Ah, about the license of icon, I really don't know much. I took it from [here](https://icons.getbootstrap.com/icons/arrow-right-short) & I think, its open source.

Reply:
The [icons](https://github.com/twbs/icons) seem to be licensed under MIT, so we would be good to go, but would need to include a copyright notice.

Reply:
ya sure, we can use this.

Reply:
Where did you find that icon? I can't seem to find it under /src/icons

Reply:
[Here](https://github.com/parse-community/parse-dashboard/blob/master/src/icons/right-outline.svg)

Reply:
Ah right, I though we had one that points into the upper right corner.